Lesson6/6_Args_sum.py

```diff
@@ -0,0 +1,35 @@
+'''
+Args sum - write a function called sum_all
+that takes any number of arguments and returns their sum.
+'''
+
+
+def sum_all(*args):
+    args_sum = 0
+    for arg in args:
+        if type(arg) in (int, float):
```

Comment:
w takich sytuacji zamiast szczegółowo sprawdzać typ można zrobić `try... catch` - próbujesz dodać, najwyżej jak nie wyjdzie to wtedy komunikat. To umożliwi dodawanie dowolnych typów, np zupełnie customowych (tak jak to umożliwia wbudowane `sum`):
```
class A:
  def __init__(self, arg):
    self.arg = arg

  def __add__(self, operand):
    return A(self.arg + operand.arg)

  def __str__(self):
    return str(self.arg)

print(sum([A(5), A(10), A(100)], A(0))) # == 115
```